Sources/Components/SaveSearchView/SaveSearchPromptView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - SaveSearchPromptViewDelegate
+
+public protocol SaveSearchPromptViewDelegate: AnyObject {
+    func saveSearchPromptView(_ saveSearchPromptView: SaveSearchPromptView, didAcceptSaveSearch: Bool)
+}
+
+// MARK: - SaveSearchPromptView
+
+public class SaveSearchPromptView: UIView {
+
+    // MARK: - Public properties
+
+    public weak var delegate: SaveSearchPromptViewDelegate?
+
+    // MARK: - Private properties
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
```

Comment:
You might want to use our `Label` class here. It supports scaling fonts which is something we want to use with all labels 👍